src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetMvcIntegration.cs

```diff
@@ -118,6 +118,13 @@ public static Scope CreateScope(dynamic controllerContext)
 
                 scope = Tracer.Instance.StartActive(OperationName, propagatedContext);
                 Span span = scope.Span;
+
+                // Fail safe to catch templates in routing values
+                resourceName =
+                    resourceName
+                       .Replace("{controller}", controllerName)
+                       .Replace("{action}", actionName);
```

Comment:
I don't think we'll need this, but to avoid breaking changes later, I added it.
It'll catch any routing templates we miss in the first resource name set.
We don't hit the route section when it's convention based, which is why I don't think it will be an issue.
I verified that "areas" are right in the url for routes as well, for when we get around to that.